### PR TITLE
realm support

### PR DESCRIPTION
**Pull request by maxhora**
- realm is loaded on start of ubuntu_server.js and passed into vm sandbox environment as global variable. Global variable is used inside modified realm index.js implementation placed into forked repo ( https://github.com/status-im/realm-js ).
- Some JS modules added back to .re-natal configuration file to resolve not found dependencies issues

### CODE REVIEW

File: src/status_im/desktop/core.cljs [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
             [re-frame.core :refer [subscribe dispatch dispatch-sync]]
             status-im.utils.db
-            ;;status-im.ui.screens.db
-            ;;status-im.ui.screens.events
-            ;status-im.ui.screens.subs
-            ;;status-im.data-store.core
+            status-im.ui.screens.db
+            status-im.ui.screens.events
+            status-im.ui.screens.subs
+            status-im.data-store.core
```
Comment (maxhora) [inline src/status_im/desktop/core.cljs:8]: uncommenting of these namespaces causes corresponding cljs files code execution

File: src/status_im/ui/screens/events.cljs [hunk 2]
```diff
@@ -274,28 +275,29 @@
...
-                          [:load-contact-groups] 
-                          [:init-discoveries]
-                          [:initialize-debugging {:address address}]
-                          [:send-account-update-if-needed]
-                          [:start-requesting-discoveries]
-                          [:remove-old-discoveries!]
-                          [:update-wallet]
-                          [:update-transactions]
-                          [:get-fcm-token]]
+                          ;[:load-processed-messages]
```
Comment (maxhora) [inline src/status_im/ui/screens/events.cljs:278]: Temporary commented out

File: src/status_im/ui/screens/events.cljs [hunk 3]
```diff
@@ -274,28 +275,29 @@
...
+                          ]
                    (seq events-after)
                    (into events-after))}))
 
 (register-handler-fx
   :check-console-chat
   (fn [{{:accounts/keys [accounts] :as db} :db} [_ open-console?]]
     (let [view (if (empty? accounts)
-                 :chat
+                 :login
```
Comment (maxhora) [inline src/status_im/ui/screens/events.cljs:300]: @vkjr the place to assign screen id into db's `:view-id` ... later it's loaded in main cljs screen

File: ubuntu-server.js [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+var util = require('util');
+var Buffer = require('buffer').Buffer;
+var realmConstructor = require("./node_modules/realm/lib/index.js");
+console.log("Loaded realmConstructor: " + realmConstructor);
+
+var DEBUG = 1;
+
+function rnUbuntuServer(readable, writable) {
+  console.reportErrorsAsExceptions = false;  // XXX:
+  var sandbox = { console: console, util: util, outerRealmConstructor: realmConstructor };
```
Comment (maxhora) [inline ubuntu-server.js:25]: `realmConstructor` is passed as `outerRealmConstructor` global variable into sandoxed env

File: src/status_im/utils/crypt.cljs [hunk 5]
```diff
@@ -18,9 +18,9 @@
   (byteArrayToHex (.digest sha-256)))
 
 (defn gen-random-bytes [length cb]
-  (.randomBytes rn-dependencies/random-bytes
+  (comment((.randomBytes rn-dependencies/random-bytes
```
Comment (vkjr) [inline src/status_im/utils/crypt.cljs]: Probably can be uncommented since random-bytes works now :)

Comment (maxhora) [reply src/status_im/utils/crypt.cljs]: right! thanks!

**Comment by maxhora** [comment]
@vkjr updated, thanks!

**Review by vkjr — APPROVED** [review]

**Pull request merged**